### ISSUE TITLE
AI Slip over mode or Clumsiness

### ISSUE THREAD

**Issue by tarkus1971** [TheGameCreators/GameGuruRepo#400]
Be great to have the enemy AI be given to be able to slip over when running very fast at the player, or turning to flank. Having a "clumsiness" factor would make the AI more human.

**Comment by tarkus1971** [comment]
The enemy soldier or NPC would be patrolling on waypoint for example, then if the player is to the side, the NPC would turn to face the player quickly and begin to run toward the player.  It os on the quick turn or during running that the NPC would kind of appear to slip or lose balance slightly, if possible also if during this "slipping" which would last for maybe a couple of seconds, the player would have opportunity to either shoot the NPC or hide from the NPC, (Switch off player detection pf the NPC during this slip too) 

It would need a very, very low randomness to this slipping or it will look unrealistic.

The randomness could be assigned a variable or editable in the Properties panel of the Entity.

**Comment by LeeBamberTGC** [comment]
Do you see this as mere intertia in the position of the enemy, or actual animation inside the character itself (sort of a cartoon like double take as they spot the player and then make a turn of the upper body followed by the lower body, etc).

**Comment by tarkus1971** [comment]
Inertia, with animation if possible, like when a real person slips, they put out a hand to reach out for support,

**Comment by LeeBamberTGC** [comment]
Yes, much easier to do ;)  Please add this observation to the AI thread posted above and see if others agree.

**Comment by tarkus1971** [comment]
OK I have included in the thread link you gave above, thanks.

**Comment by Emperor2000** [comment]
I agree, that would be very cool indeed.
